src/beautifiers/typescript-formatter.coffee

```diff
@@ -5,7 +5,7 @@ module.exports = class TypeScriptFormatter extends Beautifier
   name: "TypeScript Formatter"
   link: "https://github.com/vvakame/typescript-formatter"
   options: {
-    TypeScript: true
+    TypeScript: false
```

Comment:
Nope.

See rest of file:

```
        if options.indent_with_tabs
          opts.convertTabsToSpaces = false
        else
          opts.tabSize = options.tab_width or options.indent_size
          opts.indentSize = options.indent_size
```

Firstly, this beautifier supports `indent_with_tabs`, `tab_with`, and `indent_size`.

Secondly, I expect `TypeScript` and `TSX` languages to have the same support, unless otherwise specified.